config-xml/addon/src/main/java/org/jboss/windup/config/parser/metadata/MetadataExecuteAfterHandler.java

```diff
@@ -0,0 +1,26 @@
+package org.jboss.windup.config.parser.metadata;
+
+import static org.joox.JOOX.$;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.jboss.forge.furnace.addons.AddonId;
+import org.jboss.windup.config.exception.ConfigurationException;
+import org.jboss.windup.config.parser.ElementHandler;
+import org.jboss.windup.config.parser.NamespaceElementHandler;
+import org.jboss.windup.config.parser.ParserContext;
+import org.w3c.dom.Element;
+
+@NamespaceElementHandler(elementName = "executeAfter", namespace = "http://windup.jboss.org/v1/xml")
+public class MetadataExecuteAfterHandler implements ElementHandler<String>
+{
+
+    @Override
+    public String processElement(ParserContext handlerManager, Element element) throws ConfigurationException
+    {
+        String executeAfterId = element.getTextContent();
```

Comment:
Need to throw a validation exception if this value is missing. (Like our existing handlers.)